src/main/java/net/neoforged/neoforge/common/CommonHooks.java

```diff
@@ -987,8 +1008,8 @@ public static void onEntityEnterSection(Entity entity, long packedOldPos, long p
         NeoForge.EVENT_BUS.post(new EntityEvent.EnteringSection(entity, packedOldPos, packedNewPos));
     }
 
-    public static ShieldBlockEvent onShieldBlock(LivingEntity blocker, DamageSource source, float blocked) {
-        ShieldBlockEvent e = new ShieldBlockEvent(blocker, source, blocked);
+    public static DamageBlockEvent onDamageBlock(LivingEntity blocker, DamageContainer container, boolean originalBlocked) {
+        DamageBlockEvent e = new DamageBlockEvent(blocker, container, originalBlocked);
```

Comment:
Each one of these methods needs a javadoc detailing their intended firing site, parameters, and return values.